src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## Unreleased
 
-* Updated [Semantic Conventions](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-spans.md).
+* Updated [Semantic Conventions](https://github.com/open-telemetry/semantic-conventions/blob/v1.21.0/docs/http/http-spans.md).
   to v1.21.0. This library can emit either old, new, or both attributes.
```

Comment:
Ideally, we should have `v1.21.0` pointing to the link instead of "Semantic Conventions".

Reply:
ok. i'll get this corrected